### ISSUE TITLE
[9.7.9] 

### ISSUE THREAD

**Issue by ZachBora** [StrangeLoopGames/EcoIssues#23715]

**Comment by TwixterZ** [comment]
@ZachBora Thank you for helping us by providing more information, we have been actively investigating this issue to find the cause for a possible fix. 

Issue reference : ECO-17585

**Comment by ZachBora** [comment]
Yeah its been bugging me, I looked at the code... and my theory is the compostables item was turning into trash. Tested my theory and yep... It seems to be because it itself isn't compostable.
